crates/loro-internal/src/container/tree/tree_op.rs

```diff
@@ -42,7 +45,7 @@ impl HasLength for TreeOp {
 impl Sliceable for TreeOp {
     fn slice(&self, from: usize, to: usize) -> Self {
         assert!(from == 0 && to == 1);
-        *self
+        self.clone()
```

Comment:
maybe we need to count how often does this clone happen

Reply:
It never is called.